docs/components/component/_index.md

```diff
@@ -33,10 +33,10 @@ Most robots with a COMPONENT need at least the following hardware:
 The COMPONENT component supports the following models.
 Click on the component model name for detailed configuration information.
 
```

Comment:
Do you also want a `<!-- prettier-ignore -->` here too?

Reply:
```suggestion

<!-- prettier-ignore -->
```